drivers/volume/portworx/portworx.go

```diff
@@ -164,6 +175,16 @@ type portworxGrpcConnection struct {
 	endpoint    string
 }
 
+func isTLSEnabled() bool {
+	tls := strings.ToLower(os.Getenv(pxEnableTLS))
+	if len(tls) != 0 ||
```

Comment:
There is a bug here. This will return true for `PX_ENABLE_TLS=false`.

Reply:
oops! nice catch